src/index.html

```diff
@@ -13,11 +13,12 @@
     <main>
       <div id="main-page"></div>
     </main>
-
+    <div id="exercises-container" class="exercises-container"></div>
+   
     <load src="./partials/footer.html" />
 
     <load src="partials/modal.html" />
-
+    <load src="./partials/pagination.html"/>
```

Comment:
Please remove pagination from index

Reply:
is is not working in home still lets check together